.github/workflows/Tests.yaml

```diff
@@ -433,3 +433,87 @@ jobs:
           # We get occasional random timeouts, repeat tests to see if
           # it is a random timeout or systematic
           ctest -j2 --output-on-failure -L inputfiles --repeat after-timeout:3
+
+  arch_datastructures_tests:
```

Comment:
(picking a line at random) Does the Blaze upgrade resolve https://github.com/sxs-collaboration/spectre/issues/1807? (we can also check this once this PR is done)

Reply:
I'm not sure at what point the issue was resolved, but I've dropped the offending vectorization flag from the debug builds in one of the fixups, and the CI seems to run fine.